src/kaboom.ts

```diff
@@ -4023,6 +4011,21 @@ export default (gopt: KaboomOpt = {}): KaboomCtx => {
 						this.trigger("collideEnd", col.target, col)
 					}
 				}
+
+				if (this.isHovering()) {
```

Comment:
I'll think about this more, `isHovering()` does intersection check and is a little expensive to add one call every frame to every object with area

Reply:
I see! then I will have to think of another solution